esp8266-weather-station-color.ino

```diff
@@ -157,25 +159,29 @@ void setup() {
   // The LED pin needs to set HIGH
   // Use this pin to save energy
   // Turn on the background LED
+#ifdef TFT_LED  
   Serial.println(TFT_LED);
   pinMode(TFT_LED, OUTPUT);
   digitalWrite(TFT_LED, HIGH);    // HIGH to Turn on;
+#endif
 
   delay(100); 
   gfx.init();
   gfx.fillBuffer(MINI_BLACK);
   gfx.commit();
 
-  connectWifi();
 
 #ifdef HAVE_TOUCHPAD
   Serial.println("Initializing touch screen...");
-  if (!ts.begin()) {
+  if (!ts.begin(0x48, &Wire)) {
```

Comment:
Need to add preproc logic here to call the STMP begin. This worked with local testing:
```cpp
#ifdef TOUCH_CS
  if (!ts.begin()) {
#else
  if (!ts.begin(0x48, &Wire)) {
#endif
```